### DISCUSSION TITLE
Installation Path

### DISCUSSION THREAD

**Discussion by bdorr1105** [GTGalaxi/quick-lookup-ptrun#1] [General]
I just wanted to point out that based on the per user PowerToys Installation, the path differs from the default Program Files path. The rest of the instructions are the same, however, the path is in a hidden directory.

C:\Users\<yourusername>\AppData\Local\PowerToys\modules\launcher\Plugins

**Comment by GTGalaxi** [comment]
@bdorr1105 Thanks heaps for pointing that out, I totally over looked that! I'll update the readme and installation instructions to reflect this.
Cheers! 😄

**Reply by bdorr1105** [reply]
Absolutely, I think it's a pretty awesome tool. I tested it today and it works well

**Reply by GTGalaxi** [reply]
All updated in #2 

Thanks again for pointing it out, glad you like the tool!
